src/pages/index.js

```diff
@@ -1,22 +1,31 @@
 import { makeStyles } from "@material-ui/core/styles"
 import { Typography } from "@material-ui/core"
 import { useSelector } from "react-redux"
+import { useState } from "react"
 
 import CustomTable from "@/components/CustomTable"
+import AddNewButton from "@/components/AddNewButton"
+import RightSideDrawer from "@/components/RightSideDrawer"
 import { characterSelectors } from "@/selectors/characterSelectors"
+import AddNewCharacterForm from "@/components/AddNewCharacterForm/AddNewCharacterForm"
```

Comment:
I would just call this `CharacterForm`. The idea here is to reuse it for editing the characters as well, we will look at this later but it's good to code with that in mind